cmd/tealdbg/localLedger.go

```diff
@@ -53,6 +69,25 @@ func makeAppLedger(
 	apps := []basics.AppIndex{appIdx}
 	apps = append(apps, txn.Txn.ForeignApps...)
 
+	// first populate balances from the indexer
+	for _, acc := range accounts {
+		fmt.Println("debug hello")
+		if _, ok := balances[acc]; !ok {
+			queryString := fmt.Sprintf("%s/v2/accounts/%s?round=%v", indexerURL, acc, round)
```

Comment:
this can lead to confusion. If I specified accounts and some are fetched from indexer, some from command like args. I guess logging an info message if accounts are non-empty and we go to indexer, and no message if only indexer used

Reply:
Accounts from command args have priority over the indexer? So if accounts are non-empty we will not use the indexer.